orbit/changes/17528-automatic-install-enable-scripts

```diff
@@ -0,0 +1,2 @@
+* Scripts are now enabled by default when fleet is automatically installed
+* The fleet msi installer now has a parameter `ENABLE_SCRIPTS` to enable or disable scripts when installed through cmd
```

Comment:
Double checking: Are this changes on `fleetd` or on `fleetctl`? (If they are on `fleetctl` they should be in `changes/`)

Reply:
Sort of. While the code change is technically part of `fleetctl`, it will only hit customers when we release a new version of `fleetd`, since it the changed behaviour with regards to automatic installation will live in our hosted version of `fleetd-base.msi`. I can move the changelog but I don't want customers to get confused by thinking the change is out when it still hasn't been released. The `ENABLE_SCRIPTS` is true for both so I can add that to `/changes` as well though.

Reply:
Yeah, fleetd-base installers is a gray area between changes/ and orbit/changes/ :)